src/main/dotnet/ProjectFileParser/MSBuildCustomLocator.cs

```diff
@@ -9,7 +10,9 @@ public static void Register()
         {
             try
             {
-                MSBuildLocator.RegisterDefaults();
+				var latestVsVersion = MSBuildLocator.QueryVisualStudioInstances().OrderBy(vsInstance => vsInstance.Version).Last();
```

Comment:
the indent is strange here, please use space here

Reply:
Done